src/System.Collections.Concurrent/tests/BlockingCollectionCancellationTests.cs

```diff
@@ -21,17 +22,18 @@ public static void InternalCancellation_CompleteAdding_Negative()
             //call Take.. it should wake up with an OCE. when CompleteAdding() is called.
             Assert.Throws<InvalidOperationException>(
                () => coll1.Take());
-               // "InternalCancellation_WakingUpTake:  an IOE should be thrown if CompleteAdding occurs during blocking Take()");
+            // "InternalCancellation_WakingUpTake:  an IOE should be thrown if CompleteAdding occurs during blocking Take()");
             Assert.Throws<InvalidOperationException>(
                 () => coll1.Add(1));
-                // "InternalCancellation_WakingUpAdd:  an InvalidOpEx should be thrown if CompleteAdding occurs during blocking Add()");
+            // "InternalCancellation_WakingUpAdd:  an InvalidOpEx should be thrown if CompleteAdding occurs during blocking Add()");
             Assert.Throws<InvalidOperationException>(
                () => coll1.TryAdd(1, 1000000));  //an indefinite wait to add.. 1000 seconds.
-               // "InternalCancellation_WakingUpTryAdd:  an InvalidOpEx should be thrown if CompleteAdding occurs during blocking Add()");
+            // "InternalCancellation_WakingUpTryAdd:  an InvalidOpEx should be thrown if CompleteAdding occurs during blocking Add()");
         }
 
         //This tests that Take/TryTake wake up correctly if CompleteAdding() is called while the taker is waiting.
         [Fact]
+        [OuterLoop]
```

Comment:
Negative test cases *are* a basic scenario and should be enabled. 


I'll publish clearer guidance on this Monday, but in the mean time think of "inner loop" as dev unit tests that are
* fast and reliable enough to be run in the build
* no dependency on machine or environment state
* as part of being reliable ideally no network calls, IO, elevation of privileges, etc

This boils down to basic positive and negative test cases for the majority of public APIs. 

The idea is E2E testing; tenets such as perf, reliability, acquisition, compliance; platform specific tests (eg win7-deu); and tests that can't be mocked / are "dangerous" will run in the "outer loop" -- which will begin to come online as a rolling test system in another sprint or two.

Right now there are some tests enabled in github which strictly speaking should be moved from inner to outer when the infra comes online. That's totally OK for now -- we would much rather have test coverage enabled and running now in general while the project comes online, and we can adjust their category (ie do they run in the build vs rolling lab?) later.